js/script.js

```diff
@@ -136,4 +137,31 @@ function checkForMatch(card) {
                }, 900);
            }
     }
-}
\ No newline at end of file
+}
+
+// Resetting the game
+const resetGame = document.querySelector(".refresh");
+
+resetGame.addEventListener("click", function() {
+    // reset moves
+    moves = 0;
+    movesCount.innerText= moves;
+    
+    //reset stars
+    stars[2].classList.remove('remove-star');
+    stars[1].classList.remove('remove-star');
+    stars[0].classList.remove('remove-star');
+    
+    //reset time
+    miliSeconds = 0;
+    seconds = 0;
+    minutes = 0; 
+    timeCount.innerText = minutes + " : " + miliSeconds + seconds;
+    timeStart = false;
+    clearInterval(time);
+
+    //reset cards
```

Comment:
I'm having a little difficulty getting the cards to reset after clicking the refresh button... still working on it.